xml/src/main/resources/ehcache-core.xsd

```diff
@@ -403,4 +403,9 @@
       <xs:enumeration value="UPDATED" />
     </xs:restriction>
   </xs:simpleType>
+  <xs:simpleType name="fully-qualified-class-name-type">
```

Comment:
Minor change: rename the type to `fqcn-type` which is shorter (and I hope meaningful to a java dev)